draft-irtf-cfrg-hpke.md

```diff
@@ -1261,6 +1263,9 @@ implementation-specific detail. For example, some implementations may abort or
 panic upon a `DeriveKeyPairError` failure given that it only occurs with
 negligible probability, whereas other implementations may retry the failed
 DeriveKeyPair operation. See {{derive-key-pair}} for more information.
+As another example, some implementations of the DHKEM specified in this document
+may choose to transform `ValidationError` from `DH()` into an `EncapError`
+from `Encap()`, whereas others may choose to raise `ValidationError` unmodified.
```

Comment:
just for symmetry:
"into an `EncapError` (resp. `DecapError`) from `Encap()` (resp. `Decap()`)"